.woodpecker/test.yml

```
@@ -15,6 +15,6 @@ steps:
     commands:
       # Needed for `ct` to check which chart was edited. See https://github.com/woodpecker-ci/helm/pull/56#issuecomment-1676343140
       - git fetch origin main --unshallow --no-tags || true
```

Adding a fetch without `--unshallow` could do the trick

```suggestion
      - git fetch origin main --unshallow --no-tags || true
      - git fetch origin main --no-tags || true
```

---

I don't mind as long as we can move on :)